### ISSUE TITLE
Comment must be set for public key

### ISSUE THREAD

**Issue by durist** [scheiblingco/sshkey-tools#13]
The public key to_string() method fails if a comment isn't set. If this is intended behavior, it should at least be documented.

```
$ python3
Python 3.7.3 (default, Oct 31 2022, 14:04:00) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from sshkey_tools.keys import RsaPrivateKey
>>> rsa_priv = RsaPrivateKey.generate()
>>> rsa_pub = rsa_priv.public_key
>>> print(rsa_pub.to_string())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/dist-packages/sshkey_tools/keys.py", line 269, in to_string
    ensure_string(getattr(self, "comment", ""), encoding),
TypeError: sequence item 1: expected str instance, NoneType found
>>> rsa_pub.comment = ""
>>> print(rsa_pub.to_string())
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC39KVO4r4+ZShK4t+QGT7rV1abf9esqvDNrXOA46+KKazlWbGbvhpfKIX6CbyrWgBMfe9anA5EzeFbFn4HZxDzj7+bwGq1/dIHMSxcV4qvHfZpUbjksNYZG6c+0vbup87izimMwU46AVb9T3tFn64R1bipYP/ATOdvkp9Xsgwmktxs7y8cRBYCbGDfy2TXzTUm3Bsmurc5Mpb7cM8qdZK3zmmvVboTKKg6ETaOylFH0mjo0+nKuXUnVJGyftD5iA8htasCe4fIY79PLg3fa5jg6hhmSigIl9O7W1C2jTVr9Qgwr981ZCVZHFML58yIGu6PuRFM4QPl/ISmFHo8zib2M9KbDYeSdBNVxkaCVXR0k/XjyXhLXGolAu0HzeQBfLeh7+oZCLWOYoX1COS9PIIj2I1LKinmr9IPLwkN8a9p2KhTUZsc/XNSFChDEFjLFsq8izLJEfpxcJf8C/cJr0XFgl7IOrht7Nw/ClWOXAxs5234PB9lr8f8eSx6kztCEQjzKCNm5H3XPl7VOucubRkxytBdXrbJK4YvLPHVNnHi4Std1zTk2WbbQSC811rkKLLY7jArzLjA6HJo3BP6dGhQKpFn42+QAQeJT3Egq9rzfyWybUzFlzEqX8lbRmMTa9flxr+4OPxhm00E2c/JXegWbYUkfmXfpCR3g+sC1NFsDw== 

```

**Comment by scheibling** [comment]
Thanks for noticing this, I'll set a default for the comment variable

**Comment by scheibling** [comment]
Resolved through #14 in version 0.9.1

**Comment by scheibling** [comment]
Reopening this, noticed some cases where it's still gonna error out due to the missing comment. Will be fixed in next version

**Comment by scheibling** [comment]
Resolved (again) through #26
